Xplat/src/main/java/vazkii/botania/common/block/flower/generating/ThermalilyBlockEntity.java

```diff
@@ -70,7 +58,7 @@ public void tickFlower() {
 			ticksSinceFueled++;
 		} else if (burnTime == startBurnTime) {
 			if (ticksSinceFueled <= FAST_PROVIDE_TICKS) {
-				heat = heat < 10 ? heat + 1 : heat;
+				heat = heat < MAX_HEAT ? heat + 1 : heat;
 			} else {
 				heat = 0;
```

Comment:
Ideally I would have made it so that providing lava before the end of the cooldown would cause the thermalily to "burn" into a dead bush, but that would require either changing the logic in FluidGeneratingFlower (doable, requires some refactoring and moving of code), or use the call to getCooldownTime(false), (janky and prone to errors if other mods/addons/code calls that).

Reply:
i think just resetting the heat is an ok penalty here